geometry_msgs/Pose pose
geometry_msgs/Vector3 approach

std_msgs/Float32 grasp_width
